doc/asciidoc/faq/index.adoc

```diff
@@ -0,0 +1,167 @@
+
+== Neo4j Streams FAQ
+
+How to integrate Neo4j and Kafka::
+
+When integrating Neo4j and Kafka using Neo4j Streams plugin or Kafka Connect plugin
+is important configure just one of them and not both.
+If you need to load data from Kafka to Neo4j (and not viceversa) you can just use the Kafka Connect plugin.
+If you need to have both sink and source functionalities then you have to use the Neo4j Streams plugin.
+This is the main difference between those plugins.
+
+About CUD file format::
```

Comment:
Can you make these mini-headings into headers so that they can be link targets?  It's really nice to have a FAQ, and making things link targets will allow people to answer each other in support tickets and forums with a link to specifically the common question and its answer.

Reply:
I'll change them into sections. Thanks!